.env.example

```
@@ -1,58 +1,85 @@
+WEB_ENABLED=true
+# Bot HTTP (when WEB_ENABLED=true): Express `/api/guilds/*`, `/ws`, `/health` only — no Next.js inside the bot container.
+# The dashboard is still a normal Next.js app under `src/` (`yarn dev:web`, `yarn build:web`); run or deploy it as its own process/service.
+# WEB_PORT configures the bot/Express listener (default 3001). Next dashboard runs on 3000.
+WEB_PORT=3001
+# Verbose logging for dashboard → bot HTTP (Next terminal + bot terminal). No cookies or bodies logged.
+# BOT_API_VERBOSE=1
+# Alias (either works on Next or bot): WEB_BOT_API_VERBOSE=1
+# Better Auth: one DB + secret; two HTTP entrypoints (see src/web/lib/auth-base-config.ts).
+# BETTER_AUTH_URL = public URL of the Next dashboard (OAuth callback + cookie issuer), e.g. http://localhost:3000
+# The bot container serves plain Express /api/guilds + /ws only — it uses auth-node (no Next) but reads the same sessions.
+BETTER_AUTH_SECRET=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The BETTER_AUTH_SECRET key should go before the DEV_MODE key

(UnorderedKey)